Database/Corrections/SeasonOfDiscovery.lua

```diff
@@ -578,6 +578,9 @@ local questsToBlacklistBySoDPhase = {
     [4] = { -- SoD Phase 4 == Era Phase 1+2: level cap 60, MC, Ony, world bosses
     },
     [5] = { -- SoD Phase 5 == Era Phase 3: BWL + Darkmoon Faire
+        [4788] = true, -- The Final Tablets
+        [5065] = true, -- The Lost Tablets of Mosh'aru
+        [8181] = true, -- Confront Yeh'kinya
```

Comment:
My guess is these quests become available with ZG, as the entire quest chain is linked to it. So I'd ssay we move it to "SoD Phase 6"